js/webpack/libraries.js

```diff
@@ -30,7 +30,7 @@ module.exports = {
     // library
     'inject': ['./inject.js'],
     'web3': ['./web3.js'],
-    'parity': ['./parity.js']
+    'parity': ['./library.parity.js']
```

Comment:
Yes, the changes here

https://github.com/ethcore/parity/pull/4479/files#diff-5a5d605811790e35c3e21f577912e89eR17

and here 

https://github.com/ethcore/parity/pull/4479/files#diff-161120f1d7582520482df9363dee21acR17

Are the ones where you went wrong. You tried to convert the one into the other, instead of understanding the purpose. (Changes basically copies the one to the other - more or less) Should have kept it, moved `library.parity.js` to `index.js` (like done elsewhere) and have been done, but rather these changes got them mixed.